### ISSUE TITLE
Too frequent WARN or wrong number of seconds 

### ISSUE THREAD

**Issue by pouledodue** [apache/incubator-pulsar#1485]
#### Expected behavior

time waited should correspond to frequency of output of WARN messages
```
10:16:25.438 [main] WARN  org.apache.pulsar.client.cli.PulsarClientTool - No message to consume after waiting for 5 seconds.
10:16:30.439 [main] WARN  org.apache.pulsar.client.cli.PulsarClientTool - No message to consume after waiting for 5 seconds.
10:16:35.440 [main] WARN  org.apache.pulsar.client.cli.PulsarClientTool - No message to consume after waiting for 5 seconds.
10:16:40.441 [main] WARN  org.apache.pulsar.client.cli.PulsarClientTool - No message to consume after waiting for 5 seconds.
10:16:45.446 [main] WARN  org.apache.pulsar.client.cli.PulsarClientTool - No message to consume after waiting for 5 seconds.
```
#### Actual behavior

outputs WARN with wrong number of seconds
(it says it waited 20 seconds but the output is every 5 seconds)

```
10:16:25.438 [main] WARN  org.apache.pulsar.client.cli.PulsarClientTool - No message to consume after waiting for 20 seconds.
10:16:30.439 [main] WARN  org.apache.pulsar.client.cli.PulsarClientTool - No message to consume after waiting for 20 seconds.
10:16:35.440 [main] WARN  org.apache.pulsar.client.cli.PulsarClientTool - No message to consume after waiting for 20 seconds.
10:16:40.441 [main] WARN  org.apache.pulsar.client.cli.PulsarClientTool - No message to consume after waiting for 20 seconds.
10:16:45.446 [main] WARN  org.apache.pulsar.client.cli.PulsarClientTool - No message to consume after waiting for 20 seconds.
```
#### Steps to reproduce

```
pulsar-client consume persistent://sample/standalone/ns1/backwards \
  --subscription-name my-subscription \
  --num-messages 0
```

#### System configuration
**Pulsar version**: 2.0.0
mac

**Comment by merlimat** [comment]
Thanks for reporting this. Actually, I think it makes no sense to print that warning. I'll remove this.

**Comment by sijie** [comment]
This has been fixed by #1485
